test/integrations/destinations/ga4/dataDelivery/business.ts

```diff
@@ -0,0 +1,276 @@
+};
+
+const invalidEventNameRequest = {
+  events: [
+    {
+      name: 'campaign@details',
+      params: {
+        term: 'summer+travel',
+        medium: 'cpc',
+        source: 'google',
+        content: 'logo link',
+        campaign: 'Summer_fun',
+        campaign_id: 'google_1234',
+        engagement_time_msec: 1,
+      },
+    },
+  ],
+  user_id: 'dummyUserId',
+  client_id: 'dummyClientId',
+  non_personalized_ads: true,
+};
+
+const invalidParameterValueRequest = {
+  events: [
+    {
+      name: 'add_to_cart',
+      params: {
+        currency: 'USD',
+        value: 7.77,
+        engagement_time_msec: 1,
+        items: [
+          {
+            item_id: '507f1f77bcf86cd799439011',
+            item_name: 'Monopoly: 3rd Edition',
+            coupon: 'SUMMER_FUN',
+            item_category: 'Apparel',
+            item_brand: 'Google',
+            item_variant: 'green',
+            price: '$19',
+            quantity: 2,
+            affiliation: 'Google Merchandise Store',
+            currency: 'USD',
+            item_list_id: 'related_products',
+            item_list_name: 'Related Products',
+            location_id: 'L_12345',
+          },
+        ],
+      },
+    },
+  ],
+  user_id: 'dummyUserId',
+  client_id: 'dummyClientId',
+  non_personalized_ads: true,
+};
+
+const proxyMetdata: ProxyMetdata = {
+  jobId: 1,
+  attemptNum: 1,
+  userId: 'dummyUserId',
+  sourceId: 'dummySourceId',
+  destinationId: 'dummyDestinationId',
+  workspaceId: 'dummyWorkspaceId',
+  secret: {},
+  dontBatch: false,
+};
+
+const metadataArray = [proxyMetdata];
+
+export const testScenariosForV1API = [
+  {
+    id: 'ga4_v1_scenario_1',
+    name: 'ga4',
+    description:
+      '[Proxy v1 API] :: Test for a valid request - where the destination responds with 200 without any error',
+    successCriteria: 'Should return 200 with no error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            JSON: validRequest,
+            endpoint: 'https://www.google-analytics.com/debug/mp/collect',
+          },
+          metadataArray,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message: '[GA4 Response Handler] - Request Processed Successfully',
+            response: [
+              {
+                error: '{"validationMessages":[]}',
+                metadata: {
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  jobId: 1,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                },
```

Comment:
Use `generateMetdata` function from `testUtils.ts`

Reply:
Or use the same `metadata` variable declared above

Reply:
addressed